nautobot/extras/api/serializers.py

```diff
@@ -695,7 +718,7 @@ class JobRunResponseSerializer(serializers.Serializer):
 #
 
 
-class JobResultSerializer(serializers.ModelSerializer):
+class JobResultSerializer(CustomFieldModelSerializer, BaseModelSerializer):
```

Comment:
```suggestion
class JobResultSerializer(CustomFieldModelSerializer):
```

`CustomFieldModelSerializer` is a subclass of `BaseModelSerializer`

Reply:
Same thought here - `CustomFieldModelSerializer` really should be a mixin, IMHO.